scipy/optimize/_linprog_util.py

```diff
@@ -1372,18 +1325,19 @@ def _postsolve(x, postsolve_args, complete=False, tol=1e-8, copy=False):
     # if "complete", problem was solved in presolve; don't do anything here
     if not complete and bounds is not None:  # bounds are never none, probably
         n_unbounded = 0
-        for i, b in enumerate(bounds):
+        for i, bi in enumerate(bounds):
```

Comment:
I know it's tempting to refactor, but please try to avoid it unless it's important. Thank you!